src/components/calculator.css

```diff
@@ -8,30 +8,36 @@ body {
   margin: 0;
   font-size: 12px;
   background-color: rgb(173, 201, 201);
-}
+} */
 
 .container {
-  width: 380px;
+  width: 500px;
   display: flex;
   justify-content: center;
   align-items: baseline;
   margin: 2rem auto;
 }
 
+.calc-grid {
+  width: 80%;
+}
+
 .input {
   display: grid;
-  grid-template-columns: repeat(4, 6rem);
-  grid-template-rows: minmax(7rem, auto) repeat(5, 6rem);
-  height: 400px;
+  grid-template-columns: repeat(4, 25%);
+  grid-template-rows: minmax(3rem, auto) repeat(5, 3rem);
+  font-weight: 600;
+
+  /* height: 400px; */
```

Comment:
- [ ] Kindly please remove all lines of old commented pieces of code, so the project gets cleaner 💪

Reply:
done